build.sbt

```diff
@@ -46,12 +47,13 @@ lazy val core = crossProject(JSPlatform, JVMPlatform, NativePlatform)
       "org.scalatest" %%% "scalatest-funsuite" % "3.2.18" % Test
     ),
     // TODO: 2.13 has warnings for using Stream, but scalacheck Shrink
-    tlFatalWarningsInCi := scalaVersion.value.startsWith("2.12."),
+    tlFatalWarnings := scalaVersion.value.startsWith("2.12."),
     mimaBinaryIssueFilters ++= {
       if (tlIsScala3.value) {
         import com.typesafe.tools.mima.core._
         Seq(
-          ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.typelevel.paiges.Chunk*")
+          ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.typelevel.paiges.Chunk*"),
+          ProblemFilters.exclude[DirectMissingMethodProblem]("org.typelevel.paiges.Chunk#ChunkStream#3#Empty.this")
         )
```

Comment:
I don't think anyone should be coding against this. Not sure why it keeps having issues.

Reply:
This new error is possibly due to some change between Scala 3.3.0 and 3.3.3, maybe some default constructor for case objects got removed, idk.